index.html

```diff
@@ -77,7 +77,7 @@
               <a href="#team" class="nav-link">Speakers</a>
             </li>
             <li class="nav-item">
-              <a href="#gallary" class="nav-link">Gallary</a>
+              <a href="#gallary" class="nav-link">Gallery</a>

```

Comment:
This has already been solved. When did you fork the repo?

Please go to your fork of the project and click on `Sync` so that your fork will be up to date with the project.

Reply:
Three/four days ago

Reply:
Ok